win32/devdoc/gballoc_ll_passthrough.md

```diff
@@ -50,6 +54,31 @@ MOCKABLE_FUNCTION(, void*, gballoc_ll_malloc, size_t, size);
 
 **SRS_GBALLOC_LL_PASSTHROUGH_02_003: [** `gballoc_ll_malloc` shall call `malloc(size)` and return what `malloc` returned. **]**
 
+### gballoc_ll_malloc_2
+```c
+MOCKABLE_FUNCTION(, void*, gballoc_ll_malloc_2, size_t, nmemb, size_t, size);
+```
+
+`gballoc_ll_malloc_2` returns what `malloc` from stdlib returns when called with `nmemb`*`size`. This is useful for example when allocating a pointer to an array of `nmemb` elements each having `size` size.

```

Comment:
>malloc_2` returns what `malloc` from stdli [](http://example.com/codeflow?start=12&length=42)

I think that it's worth mentioning that this does overflow checks. After all that is why we're putting it in. That's what ./.....